docs/grpc-api.md

```diff
@@ -0,0 +1,63 @@
+# GRPC API Documentation
+
+## Table of Contents
+
+- [API](#API)
+- [Types](#Types)
```

Comment:
Is this perhaps out of date?

I ran the command from the PR on my local repo and I got a different `md` file. Here are the sections generated:
```
- [api.proto](#api.proto)
    - [GetCommitRequest](#.GetCommitRequest)
    - [GetCommitResponse](#.GetCommitResponse)
    - [GetEventsRequest](#.GetEventsRequest)
    - [GetEventsResponse](#.GetEventsResponse)
    - [GetHeaderRequest](#.GetHeaderRequest)
    - [GetHeaderResponse](#.GetHeaderResponse)
    - [GetLastRequest](#.GetLastRequest)
    - [GetLastResponse](#.GetLastResponse)
    - [GetRegistersRequest](#.GetRegistersRequest)
    - [GetRegistersResponse](#.GetRegistersResponse)
  ```